apps/aurora-portal/src/client/Project/components/ProjectCardView.tsx

```diff
@@ -26,16 +26,9 @@ export function ProjectCard({ domain, project }: ProjectCardProps) {
     >
       {/* Header: Project Name (Clickable) + PopupMenu */}
       <div className="flex justify-between items-center w-full">
-        <Link
-          href={`/${domain?.id}/projects/${project.id}/compute`}
-          className={(active) => (active ? "active" : "")}
-          onClick={(e) => {
-            e.stopPropagation()
-            setLocation(`/projects/${project.id}/compute`)
-          }}
-        >
+        <a href="#" onClick={(e) => e.preventDefault()}>
```

Comment:
I mean I see no problem with link component as a but maybe we should create our own custom one or do we need it wdyt? now with this one we lost active not active part...

Reply:
just to make it look as link